docs/commands/merge.md

```diff
@@ -0,0 +1,94 @@
+# merge
+
+## What
+
+The purpose of the `merge` command is to standardise and immediately combine
+multiple taxonomic profiles from the same tool but different samples. This produces a
+standard 'taxon table' in either wide or long format.
+
+## How
+
+To use this command, you will need multiple profiles of a single tool, the name
+of the tool, and specify an output file name.
```

Comment:
```suggestion
of the tool, and specify an output file name:
```

This is to make the sentence after with the lower case `where` makes sense. However see the suggestion below if you want to stick to proper grammar rules.